src/services/EcoBridge/OmniBridge/OmniBridge.types.ts

```diff
@@ -0,0 +1,116 @@
+import { ChainId } from '@swapr/sdk'
+import { TransactionReceipt } from '@ethersproject/abstract-provider'
+import { BigNumber } from 'ethers'
+import { EntityState } from '@reduxjs/toolkit'
+import { TokenList } from '@uniswap/token-lists'
+import { BridgeDetails, BridgingDetailsErrorMessage, SyncState } from '../EcoBridge.types'
+
+export type TokenWithAddressAndChain = {
+  chainId: ChainId
+  address: string
+}
+
+export type TokenSubgraph = {
+  address: string
+  chainId: ChainId
+  decimals: number
+  name: string
+  symbol: string
+}
+
+export type SubgraphResponse = {
+  tokens: TokenSubgraph[]
+}
+export type Token = Pick<TokenSubgraph, 'address' | 'chainId' | 'name' | 'decimals'> & {
```

Comment:
Maybe `OmnibridgeToken` ? We got tons of types with same name

Reply:
fixed [8e842e0](https://github.com/levelkdev/swapr-dapp/pull/1007/commits/8e842e0f1f1e6c49191b873027821d9ad3a07a9c)